openverse_catalog/dags/providers/provider_api_scripts/rawpixel.py

```diff
@@ -1,193 +1,282 @@
+"""
+Content Provider:       Rawpixel
+
+ETL Process:            Use the API to identify all CC-licensed images.
+
+Output:                 TSV file containing the image meta-data.
+
+Notes:                  Rawpixel has given Openverse beta access to their API.
+                        This API is undocumented, and we will need to contact Rawpixel
+                        directly if we run into any issues.
+                        The public API max results range is limited to 100,000 results,
+                        although the API key we've been given can circumvent this limit.
+                        https://www.rawpixel.com/api/v1/search?tags=$publicdomain&page=1&pagesize=100
+"""
+import base64
+import hmac
+import html
 import logging
-from urllib.parse import parse_qs, urlparse
+import re
+from urllib.parse import urlencode
 
-import requests
-from common.licenses import get_license_info
+from airflow.models import Variable
+from common import constants
+from common.licenses import NO_LICENSE_FOUND, get_license_info
```

Comment:
Pycharm complains here:

> Cannot find reference 'NO_LICENSE_FOUND' in `'__init__.py'`

Interesting that it still works 🤔

Reply:
How odd.... because it's definitely there 😅 

https://github.com/WordPress/openverse-catalog/blob/9976c1f42b9ed1621be33c8d96a7fc041636cebe/openverse_catalog/dags/common/licenses/__init__.py#L9-L8